src/__tests__/components/EditorSideBar/EditorSideBarContent.test.tsx

```diff
@@ -1,9 +1,9 @@
-import { render } from "@testing-library/react";
+import renderWithMockedProvider from "../../utils";
 import EditorSideBarContent from "@/components/EditorSideBar/EditorSideBarContent";
 
 describe("EditorSideBarContent", () => {
   it("Should render closing button", () => {
-    const { getByRole } = render(
+    const { getByRole } = renderWithMockedProvider(
```

Comment:
I think you can use sreen.getByRole here

Reply:
It has been updated, thanks Robert~